scripts/cross_compile.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+TYPE=$1
+GCC=$2
+GH_TOKEN=$3
+
+if [[ "${TYPE}" == "armv6" ]]; then
+  ARCH="arm-linux-gnueabi"
+elif [[ "${TYPE}" == "armv7" ]]; then
+  ARCH="arm-linux-gnueabihf"
+else
+  exit 1
+fi
+
+# Update dependencies
+sudo apt-get -qq update
+sudo apt-get install -y "gcc-${GCC}-${ARCH}" "g++-${GCC}-${ARCH}"
+
+export CC="${ARCH}-gcc-${GCC}"
+export CXX="${ARCH}-g++-${GCC}"
+export STRIP="${ARCH}-strip"
+export ZMQ_BUILD_OPTIONS="--host=${ARCH}"
+
+echo "Build zeromq.js for ${ARCH} with gcc ${GCC}"
+
+if [[ -z $3 ]]; then
+  npm install "--arch=${ARCH}"
+else
+  PREBUILD_OPTS=" -u ${GH_TOKEN}"
+fi
+
+# Set architecture to arm because prebuild does not differ between arm versions at the moment
```

Comment:
Right, this is just for testing purposes and not final. Prebuild does not differ between arm versions, it only knows the `platform.arch` value.
I will patch this in prebuild (https://github.com/mafintosh/prebuild/issues/174) or in this package, I don't know yet how I'll do this.